airflow/sensors/time_delta.py

```diff
@@ -64,7 +64,7 @@ class TimeDeltaSensorAsync(TimeDeltaSensor):
     def execute(self, context: Context):
         target_dttm = context["data_interval_end"]
         target_dttm += self.delta
-        self.defer(trigger=DateTimeTrigger(moment=target_dttm), method_name="execute_complete")
+        self.defer(trigger=DateTimeTrigger(moment=target_dttm))
 
     def execute_complete(self, context, event=None):
```

Comment:
Can't we toss this too then?

Reply:
yes we can i was thinking backcompat though? i dunno